backstory/src/main/java/com/google/sps/story/StoryManagerImpl.java

```diff
@@ -0,0 +1,149 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.story;
+
+import com.google.api.client.http.ByteArrayContent;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.IdTokenCredentials;
+import com.google.auth.oauth2.IdTokenProvider;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Scanner;
+import org.json.JSONObject;
+
+/**
+ * Object which generates text through interface with Cloud Contained
+ * GPT-2 Model and indication of associated parameters.
+ */
+public final class StoryManagerImpl implements StoryManager {
+  /**
+   * prefix - String to indicate basis for text generation
+   * maxTextLength - Maximum character length of generation
+   * temperature - The volatility of topics and style in generation.
+   */
```

Comment:
Can this JavaDoc be split into one per variable? There's tooling that associates JavaDoc to members, so all of this doc would end up being associated with the prefix variable.

They can be single line to conserve space and typing, if that's the concern.

```
/** String to indicate basis for text generation. */
```